app/controllers/foreman_rh_cloud/react_controller.rb

```diff
@@ -0,0 +1,7 @@
+module ForemanRhCloud
+  class ReactController < ::ApplicationController
+    def inventory_upload
+      render 'inventory_upload', :layout => false
```

Comment:
I think you don't have to explicitly call `render` here. `inventory_upload.html.erb` will be called implicitly

Reply:
I tried, though we need to specify in some way the `:layout => false` for not using core page layout.. it didn't work for me